sharding/colocation.rst

```diff
@@ -90,12 +90,11 @@ Across all shards of the event table (Q2):
 
   SELECT page_id, count(*) AS count
   FROM event
-  WHERE page_id IN (ARRAY[...page IDs from first query...]) AND
-        tenant_id = 6 AND
-        payload->>'time' >= now() - interval '1 week'
+  WHERE page_id = ANY(ARRAY[...page IDs from first query...])
```

Comment:
@begriffs Have you tested this query to check that it works as expected (and makes sense in the context of this section)?